ui/app/templates/vault/cluster/access/identity/show.hbs

```diff
@@ -42,4 +30,18 @@
     </ul>
   </nav>
 </div>
+<Toolbar>
+  <ToolbarActions>
+    {{#unless (or (and (eq model.identityType "group") (eq model.type "internal")) model.alias)}}
+      {{#link-to "vault.cluster.access.identity.aliases.add" (pluralize model.identityType) model.id class="button has-icon-right is-ghost is-compact" data-test-entity-create-link=true}}
```

Comment:
Why not use `ToolbarButton` here? 🤔 What do you think about calling it `ToolbarLink` instead? Just noticing that it's an anchor at the end not a button.

Reply:
We call it a button in the design system because it's styled like a ghost button and I'd want them to match, but I might not be opposed to renaming it there.

Reply:
Oops, didn't answer that first question. I think I just missed this one somehow. Thanks!

Reply:
Oh just realized that this could be the reason this is blue instead of black as well (see the last question on my review comment). Re: naming - will there be any cases where we want it to be a <button> ? (I guess there aren't any yet, so we can probably punt until later).